openhands/server/routes/manage_conversations.py

```diff
@@ -725,3 +767,68 @@ def add_experiment_config_for_conversation(
         return True
 
     return False
+
+
+@app.get('/microagent-management/conversations')
+async def get_microagent_management_conversations(
+    page_id: str | None = None,
+    limit: int = 20,
+    selected_repository: str | None = None,
```

Comment:
```suggestion
    selected_repository: str,
```

should this ever be None? it seems like we never return any conversations if it is, maybe the FE should disable the query if a repo isn't selected